typescript-babel-combination/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "example-typescript-babel-combination",
+  "version": "1.0.0",
+  "description": "TypeScript-Babel Combination",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "check-types": "tsc",
+    "compile": "babel src -d lib --extensions '.ts,.tsx'",
```

Comment:
Could we add a note about this `--extensions` into the README too? Perhaps both can go under a `Handling extensions in Babel` sub-heading? One for testing ts+babel with mocha and One for compiling ts+babel (like this).

Compilation one something like `"Babel looks for .js files by default, and sadly this is not configurable within the Babel config file. It is recommended to be set via Babel CLI"`. 
Its really useful to know these.

Reply:
I have updated REAMDE as you told me. I don't know if I have fully understood your suggestion. Leave me comments if I have to add or edit something :)

Reply:
Yep thats perfect.